modules/deploy/pages/deployment-option/self-hosted/manual/sizing.adoc

```diff
@@ -74,7 +74,7 @@ Redpanda is basically a distributed transaction log with well-understood access
 
 Your best storage solution for your workload depends on your performance and data retention requirements.
 
-If high throughput and low latency is most important, then use locally-attached NVMe SSD disks. This is also a good option in the cloud. Just remember that in the cloud, local disks are ephemeral, so data is wiped when an instance is restarted. An alternative in the cloud is to use SSD-backed object storage to persist data between instance restarts. Most cloud providers have options for guaranteeing throughput and provisioned IOPS performance of network-attached storage, although network-attached storage almost always exhibits slightly higher tail latencies than direct-attached storage.
+If high throughput and low latency is most important, then use locally attached NVMe SSD disks. This is also a good option in the cloud. Just remember that in the cloud, local disks are ephemeral, so data is wiped when an instance is restarted. An alternative in the cloud is to use SSD-backed object storage to persist data between instance restarts. Most cloud providers have options for guaranteeing throughput and provisioned IOPS performance of network-attached storage, although network-attached storage almost always exhibits slightly higher tail latencies than direct-attached storage.
```

Comment:
again--not sure why you are removing the hyphen here... locally-attached NVMe...

Reply:
Note that in the cloud, local disks are ephemeral, so data is wiped when an instance is restarted. An alternative is to use SSD-backed object storage to persist data between instance restarts.